specification/columns/capacityreservationid.md

```diff
@@ -0,0 +1,37 @@
+# Capacity Reservation ID
+
+A Capacity Reservation ID is the identifier assigned to a [*capacity reservation*](#glossary:capacity-reservation) by the provider. Capacity Reservation ID is commonly used for scenarios to allocate charges for capacity reservation usage.
+
+The CapacityReservationId column adheres to the following requirements:
+
+* CapacityReservationId MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
+* CapacityReservationId SHOULD NOT be null when a charge is related to a capacity reservation.
+* CapacityReservationId MUST NOT be null when a charge represents the unused portion of a *capacity reservation*.
```

Comment:
If we change the previous requirement to a MUST, we don't need this. Fwiw, CommitmentDiscountId only has the previous statement, which technically covers used and unused. I don't have a fundamental issue with it. Just calling it out since it's a little duplicative. It also doesn't make sense to require unused but not used. If one can't be provided, then that should just be a documented gap rather than a "by design" limitation.
```suggestion
```

Reply:
I think it's important to make it possible to be conformant without having to label used ODCRs charges.